Code/Framework/AzFramework/Platform/Linux/AzFramework/Input/Buses/Notifications/RawInputNotificationsBus_xcb.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/EBus/EBus.h>
+
+#include <xcb/xcb.h>
+
+namespace AzFramework
+{
+    ////////////////////////////////////////////////////////////////////////////////////////////////
+    //! EBus interface used to listen for raw xcb input as broadcast by the system. Applications
+    //! that want xcb events to be processed by the AzFramework input system must broadcast all
+    //! input events received by the native input handle, which is the lowest level we can get input.
+    //!
+    //! It's possible to receive multiple events per button/key per frame, and it is likely that
+    //! xcb input events will not be dispatched from the main thread, so care should be taken to
+    //! ensure thread safety when implementing event handlers that connect to this xcb event bus.
+    //!
+    //! This EBus is intended primarily for the AzFramework input system to process raw input events.
+    //! Most systems that need to process input should use the generic AzFramework input interfaces,
+    //! but if necessary it is perfectly valid to connect directly to this EBus for xcb events.
+    class RawInputNotificationsXcb : public AZ::EBusTraits
+    {
+    public:
+        static const AZ::EBusAddressPolicy AddressPolicy = AZ::EBusAddressPolicy::Single;
+
+        static const AZ::EBusHandlerPolicy HandlerPolicy = AZ::EBusHandlerPolicy::Multiple;
+
+        virtual ~RawInputNotificationsXcb() = default;
+
+        virtual void OnRawInputEvent(xcb_generic_event_t* event) = 0;
```

Comment:
Can we rename to 'OnEvent' or 'HandleEvent'?

Reply:
Done.